sdk/storage/storage-file/ChangeLog.md

```diff
@@ -1,18 +1,44 @@
 # Changelog
 
+2019.07 Version 11.0.0-preview.1
+
+- [Breaking] Client types are renamed from *URL to *Client.
+  - ServiceURL, ShareURL, DirectoryURL and FileURL to ServiceClient to FileServiceClient, ShareClient, DirectoryClient and FileClient respectively.
+- [Breaking] Aborter parameters are now moved into option bags.
+  - `abortSignal` attrubute(optional) in the option-bag of respective module has to be utitlized for the `Aborter.timeout(<milliseconds>)` functionality.
+  - `Aborter.none` is the default value.
+- [Breaking] I- prefixes are removed from interface names.
+  - Example- `IDirectoryCreateOptions` is updated to `DirectoryCreateOptions`, the new names have to be used.
+- [Breaking] The static methods to create client types are removed. The functionality is moved into new instance methods added to the parent clients.
+- [Breaking] The telemetry strings have been updated.
+  - `Azure-Storage/${SDK_VERSION}` is updated to `azsdk-js-storageblob/${SDK_VERSION}`.
+- [Breaking] `withPipeline()` method is removed.
+- Async iterators with pagination support are added for listing methods
+  - `listFilesAndDirectories()` and
+  - `listShares()`
```

Comment:
link to the sample to showing the usage ?

Reply:
You can mention that they can go to samples folder. But the link maybe outdated in the future.